src/caches/beliefpropagationcache.jl

```diff
@@ -33,17 +32,15 @@ default_partitioned_vertices(ψ::AbstractITensorNetwork) = group(v -> v, vertice
 function default_partitioned_vertices(f::AbstractFormNetwork)
   return group(v -> original_state_vertex(f, v), vertices(f))
 end
-default_cache_update_kwargs(cache) = (; maxiter=20, tol=1e-5)
+default_cache_update_kwargs(cache) = (; maxiter=25, tol=1e-8)
 function default_cache_construction_kwargs(alg::Algorithm"bp", ψ::AbstractITensorNetwork)
   return (; partitioned_vertices=default_partitioned_vertices(ψ))
 end
 
-function message_diff(
-  message_a::Vector{ITensor}, message_b::Vector{ITensor}; message_norm=default_message_norm
-)
+function message_diff(message_a::Vector{ITensor}, message_b::Vector{ITensor})
```

Comment:
An optimization here would be to implement computing the overlaps and norms as network contractions, but that can be left for future work. Could you add a comment about that as a reminder? The current code will probably be slow for messages that are networks. For example if the messages are themselves MPS, the current code would contract the MPS separately and then overlap the result instead of doing a proper inner product of the MPS.

Reply:
Sure I am happy to add a comment about dealing with overlaps / norms as network contractions.